test/unit_tests/util/MomentSpec.js

```diff
@@ -18,9 +18,7 @@
  */
 
 import moment from 'moment';
-/* eslint-disable no-unused-vars */
-import {isSameDay, isSameMonth, isCurrentYear, isToday} from '../../../src/script/util/moment';
-/* eslint-enable no-unused-vars */
```

Comment:
These are used, wo why disable `no-unused-vars`?

Reply:
Probably something that evolved organically... There might have been a reason at some point (probably for other files...)